internal/utils/config.go

```diff
@@ -99,6 +100,24 @@ func MakeConfig() (*types.Config, error) {
 	ldapUserFilter := getEnv("LDAP_USERFILTER", "(cn=%s)")
 	tenant := strings.ToLower(getEnv("TENANT", KubiTenantUndeterminable))
 
+	podSecurityAdmissionEnforcement, errPodSecurityAdmissionEnforcement := podSecurity.ParseLevel(strings.ToLower(getEnv("PODSECURITYADMISSION_ENFORCEMENT", string(podSecurity.LevelRestricted))))
```

Comment:
nit: no need to give the default, parseLevel would do it for us. 

```
podSecurityAdmissionEnforcement, errPodSecurityAdmissionEnforcement := podSecurity.ParseLevel(strings.ToLower(os.getEnv('PODSECURITYADMISSION_ENFORCEMENT')))
```

No need to propose a new change for that.

Reply:
La valeur par défaut n'est en effet pas utile, mais m'empêche d'utiliser la fonction getEnv et donc d'utiliser la fonction LookupEnv mais cette fonction à deux retours ce qui va un peu complexifier le code si on doit l'utiliser.
Donc sois je laisse en l'état soit je dois créer une variable temporaire pour stocker uniquement le retour de la valeur de LookupEnv et non le boolean

Reply:
laissons a l'etat, comme je mentionnais ci-dessus.

note que os.getenv peut etre utilisée (pas besoin du LookupEnv): https://pkg.go.dev/os#Getenv .